great_expectations/data_context/store/tuple_store_backend.py

```diff
@@ -542,18 +543,9 @@ def list_keys(self):
         for blob in gcs.list_blobs(self.bucket, prefix=self.prefix):
             gcs_object_name = blob.name
             gcs_object_key = os.path.relpath(gcs_object_name, self.prefix,)
-            if self.filepath_prefix and not gcs_object_key.startswith(
-                self.filepath_prefix
-            ):
-                continue
-            elif self.filepath_suffix and not gcs_object_key.endswith(
-                self.filepath_suffix
-            ):
-                continue
-            key = self._convert_filepath_to_key(gcs_object_key)
+            key = os.path.join(self.prefix, gcs_object_key)
```

Comment:
Good catch. Hadn't noticed that .list_blobs is already getting prefix arg, so this check isn't needed.

Reply:
Oh, actually, just checked and I think we actually do still need this. `prefix` is distinct from `filepath_prefix`.

Reply:
This is already back in.